src/utils.test.js

```diff
@@ -120,7 +120,22 @@ const getPrettierOptionsFromESLintRulesTests = [
   },
   {rules: {'max-len': 2}, options: {printWidth: 80}},
   {rules: {'comma-dangle': [2, 'never']}, options: {trailingComma: 'none'}},
-  {rules: {'comma-dangle': [2, 'always']}, options: {trailingComma: 'all'}},
+  {rules: {'comma-dangle': [2, 'always']}, options: {trailingComma: 'es5'}},
```

Comment:
If you want this library to match the default eslint values, then this would have to be a breaking change. As it is now, this test is technically "wrong".

My use case is already possible by simply using the object syntax with all the default eslint values. These configs are equivalent in eslint, but produce different prettier configs:

```js
// becomes trailingComma: 'all'
{
  'comma-dangle': [2, 'always-multiline'],
}

// becomes trailingComma: 'es5'
{
  'comma-dangle': [
    2,
    {
      arrays: 'always-multiline',
      objects: 'always-multiline',
      imports: 'always-multiline',
      exports: 'always-multiline',
      functions: 'ignore',
    },
  ]
}
```

This PR was more to match eslint's defaults than to support a new use case. If breaking changes aren't feasible, I think there should be something in the docs regarding this discrepancy.